### PR TITLE
apply seo recommendations

### PR DESCRIPTION
**Pull request by NansiYancheva**
Related to [#8914](https://github.com/telerik/blazor/issues/8914)

### CODE REVIEW

**Review by dimodi — APPROVED** [review]

File: components/datepicker/overview.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Overview
+title: Blazor DatePicker Overview | Telerik UI for Blazor
```
Comment (yordan-mitev) [inline components/datepicker/overview.md:2]: This is the title displayed in the lefthand navigation. It must remain unchanged as it doesn't affect SEO in any way.
```suggestion
title: Overview
```

File: components/datepicker/overview.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 page_title: Date Picker Overview
```
Comment (yordan-mitev) [inline components/datepicker/overview.md:3]: This is the SEO title that you could change. However, note the following:
- This title is generated by a template that looks like this: "Blazor" + page_title + - Telerik UI for Blazor.
- The currently rendered title is "Blazor Date Picker Overview - Telerik UI for Blazor".
- There is only one thing that you could change and this is the component name: Date Picker -> DatePicker.
- Changing the separator (from `-` to `|`) is not an option as it is part of the template and we have an agreement with the SEO team that we won't change the separators. If they've requested this change, this is probably a mistake on their part.

```suggestion
page_title: DatePicker Overview
```

File: components/dialog/predefined.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Predefined Dialogs
+title: Blazor Dialogs - Alert, Confirm, Prompt | Telerik UI for Blazor
```
Comment (yordan-mitev) [inline components/dialog/predefined.md:2]: Do not change this title—it doesn't affect SEO.
```suggestion
title: Predefined Dialogs
```

File: components/dialog/predefined.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
 page_title: Predefined Dialogs - Alert, Confirm, Prompt
```
Comment (yordan-mitev) [inline components/dialog/predefined.md:3]: Consider the `page_title` (SEO title) template when implementing requirements by the SEO team.
```suggestion
page_title: Dialogs - Alert, Confirm, Prompt
```

File: components/dialog/predefined.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ position: 2
 
 # Predefined Dialogs - Alert, Confirm, Prompt
 
-Telerik UI for Blazor provides styled substitutes to the standard confirm, alert and prompt dialogs. They match the Theme of the components to make it obvious to the user that the modal dialog is coming from your application.
+Telerik UI for Blazor provides styled substitutes to the standard confirm, alert and prompt dialogs. The Blazor popup messages match the Theme of the components to make it obvious to the user that the modal dialog is coming from your application.
```
Comment (yordan-mitev) [inline components/dialog/predefined.md:13]:
```suggestion
Telerik UI for Blazor provides styled substitutes to the standard confirm, alert, and prompt dialogs. These Blazor popup messages match the Theme of the components to make it obvious to the user that the modal dialog is coming from your application.
```

File: components/dialog/predefined.md [hunk 6]
```diff
@@ -10,7 +10,7 @@ position: 2
...
 
 To use these dialogs, receive a cascading parameter of type `Telerik.Blazor.DialogFactory`. It exposes the methods you can use in your method calls.
```
Comment (yordan-mitev) [inline components/dialog/predefined.md:15]: When calling one thing different names, we introduce ambiguity, so make sure the reader is still able to understand what we're referring to.
```suggestion
To use these dialogs (popup messages), receive a cascading parameter of type `Telerik.Blazor.DialogFactory`. It exposes the methods you can use in your method calls.
```

File: components/menu/navigation.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Navigation
+title: Blazor Navigation Menu | Telerik UI for Blazor
```
Comment (yordan-mitev) [inline components/menu/navigation.md:2]:
```suggestion
title: Navigation
```

File: components/menu/navigation.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
...
 page_title: Menu - Navigation
```
Comment (yordan-mitev) [inline components/menu/navigation.md:3]:
```suggestion
page_title: Navigation Menu
```

File: knowledge-base/gauge-pointer-tooltip.md [hunk 9]
```diff
@@ -35,7 +35,7 @@ How can I add a Tooltip to the Blazor Arc or Radial Gauge? I want to show a Tool
 To add a Tooltip for the pointer of the Gauge:
 
 1. Set the [`Color` of the Gauge Pointer]({%slug arc-gauge-pointers%}#color).
-1. Declare an instance of [`TelerikTooltip`]({%slug tooltip-overview%}).
+1. Declare an instance of [`Telerik UI for Blazor Tooltip`]({%slug tooltip-overview%}).
```
Comment (yordan-mitev) [inline knowledge-base/gauge-pointer-tooltip.md:38]: Use `monospace` only when something is considered inline code.
```suggestion
1. Declare an instance of `TelerikTooltip` ([Telerik UI for Blazor Tooltip])({%slug tooltip-overview%}).
```

**Review by yordan-mitev — CHANGES_REQUESTED** [review]
Please fix the issues with the `title` and `page_title` fields.

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**